MovieNight/ActorViewController.swift

```diff
@@ -63,9 +65,25 @@ class ActorViewController: UIViewController, UITableViewDataSource, UITableViewD
         // Assign the indexPath to the button's tag so the actor selected can be retreived when the button is tapped
         cell.loveButton.tag = indexPath.row
         
+        if selectedindexPaths.contains(indexPath.row) {
+            selectImageButton(button: cell.loveButton)
+        } else {
+            deselectImageButton(button: cell.loveButton)
+        }
+        
         return cell
     }
     
+    func selectImageButton(button: UIButton) {
```

Comment:
It's nice that you created these two methods :) because you avoid code duplication and it also improves readability.